### DISCUSSION TITLE
setTimeout not working in Vercel Deployment

### DISCUSSION THREAD

**Discussion by ebulku** [vercel/next.js#72542] [Help]
### Summary

I am trying to simulate a slow data fetch to test the skeletons, but the setTimeout() function is not working in the deployment, but locally is working fine.

`await new Promise((resolve) => setTimeout(resolve, 3000));`

### Additional information

I have tried both with `experimental_ppr` and without it.

### Example

https://nextjs.org/learn/dashboard-app/static-and-dynamic-rendering#simulating-a-slow-data-fetch

**Comment by robojeff** [comment]
I'm experiencing the exact same issue.  At the conclusion of Chapter 9 of the NextJS tutorial, the delay is working locally, but when I push to Vercel it doesn't have any effect and the revenue chat is rendered immediately.
